__docs__/wonder-blocks-accordion/accordion.stories.tsx

```diff
@@ -517,6 +522,57 @@ export const LongSections: StoryComponentType = {
     },
 };
 
+/**
+ * This is an example of an Accordion with a dropdown within each section.
+ * This demonstrates how the accordion keyboard interactions do not interfere
+ * with the dropdown's keyboard interactions.
+ */
+const items = [
+    <OptionItem label="Banana" value="banana" key={0} />,
+    <OptionItem label="Strawberry" value="strawberry" disabled key={1} />,
+    <OptionItem label="Pear" value="pear" key={2} />,
+];
+export const WithDropdown: StoryComponentType = {
+    render: function Render() {
+        const [value, setValue] = React.useState<any>(null);
+        const [singleOpened, setSingleOpened] = React.useState(false);
+
+        const [values, setValues] = React.useState<any>([]);
+        const [multiOpened, setMultiOpened] = React.useState(false);
+
+        return (
+            <Accordion animated={true}>
+                <AccordionSection header={`Single Select`}>
+                    {/* Adding height because overflow hidden in sections. */}
+                    <View style={singleOpened && {height: 200}}>
+                        <SingleSelect
+                            placeholder="Select an option"
+                            selectedValue={value}
+                            onChange={setValue}
+                            opened={singleOpened}
+                            onToggle={setSingleOpened}
+                        >
+                            {items}
+                        </SingleSelect>
+                    </View>
+                </AccordionSection>
+                <AccordionSection header={`Multi Select`}>
+                    <View style={singleOpened && {height: 200}}>
+                        <MultiSelect
+                            selectedValues={values}
+                            onChange={setValues}
+                            opened={multiOpened}
+                            onToggle={setMultiOpened}
+                        >
+                            {items}
+                        </MultiSelect>
+                    </View>
+                </AccordionSection>
+            </Accordion>
+        );
+    },
```

Comment:
**suggestion:** Ah one extra thing.... as this story doesn't need to be snapshotted by Chromatic b/c it's more for interactive purposes (keyboard navigation), I think we can skip it from Chromatic.

```suggestion
    },
    parameters: {
        chromatic: {
            // Disabling because this doesn't test anything visual.
            disableSnapshot: true,
        },
    }
```